<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit</class>
 <widget class="QDialog" name="Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
  background-color: rgb(149, 249, 255);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>18</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Amount of credit:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>80</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Credit term:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>140</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Interest rate:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>210</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Type of monthly payments:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Annuity">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>230</y>
     <width>139</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Annuity</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="Differentiated">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>260</y>
     <width>139</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Differentiated</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calculCredit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 255, 255);
  border: 1px solid grey;
  border-radius: 10px;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}

</string>
   </property>
   <property name="text">
    <string>СALCULATE</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="payment">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
  background-color: white;
  border: 1px solid gray;
}</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextEditable</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>182</x>
     <y>44</y>
     <width>15</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>21</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>mon</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>21</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="amount_of_credit">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>40</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
  background-color: white;
}</string>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="maximum">
    <double>1000000000000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1000.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="amount_months">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>100</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
	background-color:white;
}</string>
   </property>
   <property name="maximum">
    <number>600</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="interest_rate">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>161</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
	background-color:white;
}</string>
   </property>
   <property name="maximum">
    <double>99.989999999999995</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Monthly payment:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="overpayment">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>103</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
  background-color: white;
  border: 1px solid gray;
}</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextEditable</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>72</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Overpayment on loan:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>138</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Total payout:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="totalPayment">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>167</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
  background-color: white;
  border: 1px solid gray;
}</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextEditable</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>476</x>
     <y>44</y>
     <width>15</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>476</x>
     <y>108</y>
     <width>15</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>476</x>
     <y>170</y>
     <width>15</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
  <widget class="QListWidget" name="mon_pay">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>11</y>
     <width>331</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget {
	background-color: white;
	  border: 1px solid gray;
}</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="start_date">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateEdit{
	background-color: white;
}</string>
   </property>
   <property name="minimumDate">
    <date>
     <year>2023</year>
     <month>10</month>
     <day>1</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>dd/MM/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="currentSectionIndex">
    <number>0</number>
   </property>
   <property name="date">
    <date>
     <year>2023</year>
     <month>10</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>290</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Loan start date:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calculCredit_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>332</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 255, 255);
  border: 1px solid grey;
  border-radius: 10px;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}

</string>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
